processing/src/main/java/io/druid/segment/filter/InFilter.java

```diff
@@ -97,4 +99,28 @@ public boolean apply(String input)
         }
     );
   }
+
+  @Override
+  public ValueMatcher makeMatcher(
+      BitmapIndexSelector selector,
+      ValueMatcherFactory valueMatcherFactory,
+      RowOffsetMatcherFactory rowOffsetMatcherFactory
+  )
+  {
+    if (supportsBitmapIndex(selector)) {
+      return rowOffsetMatcherFactory.makeRowOffsetMatcher(getBitmapIndex(selector));
+    } else {
+      return makeMatcher(valueMatcherFactory);
+    }
+  }
+
+  @Override
+  public boolean supportsBitmapIndex(BitmapIndexSelector selector)
+  {
+    if(selector.getBitmapIndex(dimension) == null) {
```

Comment:
[nit] `return selector.getBitmapIndex(dimension) != null` is more readable to my eyes